hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestOmSnapshotManager.java

```diff
@@ -95,9 +95,9 @@ public class TestOmSnapshotManager {
   private File f1File;
 
   @BeforeEach
-  public void init() throws Exception {
+  public void init(@TempDir Path pathTestDir) throws Exception {
     OzoneConfiguration configuration = new OzoneConfiguration();
-    testDir = GenericTestUtils.getRandomizedTestDir();
+    testDir = pathTestDir.toFile();
```

Comment:
`testDir` can be annotated directly, then we don't need the parameter and the assignment.

Reply:
Done. thanks.